src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/Airport
```

Comment:
The changes in this file are not necessarily related to the header.
It's fine for now, but I wanted to point this out just for you to be careful in the future. 😉